### DISCUSSION TITLE
Any plans for documentation?

### DISCUSSION THREAD

**Discussion by ForsythsGambit** [MCPI-Devs/MCPIL#16] [Q&A] [answered]
Is there any form of documentation available, plans or a need to do so? I'd love to be able to lend a hand in any way I can!
If someone could help me get a basic understanding of the structure of the project I'd like to help with writing the wiki.

**Comment by ForsythsGambit** [comment, marked as answer]
Nevermind,if anyone else is looking for it here you go:https://wiki.mcpirevival.tk/wiki/Main_Page

**Reply by torralbaa** [reply]
Hello! And excuse me for for the late response!

Currently in the Wiki (your link :+1:) we have some general information about MCPI and related projects. You can also check [MCPI.tk](https://mcpi.tk) our "main page". Since we're an open project, we don't have everything centralized, so maybe you could find useful information in other less-conventional places, like Discord or Gitter, but MCPI.tk is a very accurate index.

Thanks for your interest in the project! :D
